lib/storage/shared.js

```diff
@@ -487,6 +490,54 @@ async function issueNoteChallenge(client, requestParams, body = {}) {
   return request.json()
 }
 
+async function createGroupRequest(
+  crypto,
+  data,
+  encryptionKeyPair,
+  recipientEncryptionKey
```

Comment:
I don't believe this field is needed. The function should generate the new group keypair